open-ce/conda_env_file_generator.py

```diff
@@ -0,0 +1,108 @@
+"""
+*****************************************************************
+Licensed Materials - Property of IBM
+(C) Copyright IBM Corp. 2020. All Rights Reserved.
+US Government Users Restricted Rights - Use, duplication or
+disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+*****************************************************************
+"""
+
+import re
+import os
+import yaml
+import utils
+
+class CondaEnvFileGenerator():
+    """
+    The CondaEnvData class holds all of the information needed to generate a conda
+    environment file while can be used to create a conda environment from the built packages.
+    """
+
+    #pylint: disable=too-many-instance-attributes,too-many-arguments
+    def __init__(self,
+                 python_versions=utils.DEFAULT_PYTHON_VERS,
+                 build_types=utils.DEFAULT_BUILD_TYPES,
+                 channels=None,
+                 output_folder=None,
+                 env_file_prefix=utils.CONDA_ENV_FILENAME_PREFIX,
+                 ):
+        self.python_versions = python_versions
```

Comment:
It might be cleaner to add the parse_arg_list function here, instead of where self.python_versions is being used. (Same for build_types).